stl/inc/yvals_core.h

```diff
@@ -283,6 +283,7 @@
 // P0288R9 move_only_function
 // P0401R6 Providing Size Feedback In The Allocator Interface
 // P0448R4 <spanstream>
+// P0627R6 Function To Mark Unreachable Code

```

Comment:
Thanks for Title Casing this! :heart_eyes_cat: I'd like to go with just `P0627R6 unreachable()` for this paper (but you couldn't have known as you created this PR faster than I filed #2531 :joy_cat:).

The idea behind this "cleaned-up title" convention is that when a paper introduces a single new thing, it's often simplest to just use that as the title.

Reply:
That's Casey suggested to Title Cased this. I put just `P0627R6 unreachable()`, copying `P1132R7 out_ptr(), inout_ptr()`. I'll change that back.